### DISCUSSION TITLE
Umi4.0 怎么获取动态路由

### DISCUSSION THREAD

**Discussion by lecd** [umijs/umi#9619] [General]
路由配置  比如：
{
    name: '产品管理',
    path: '/product/manage/:id',
    component: './Product/Manage/[id]',
    access: 'normalRouteFilter',
    hideInMenu: true,
  }

我在页面怎么获取到值 /product/manage/:id 而不是/product/manage/23

**Comment by fz6m** [comment]
在 [useRouteData](https://umijs.org/docs/api/api#useroutedata) 里。
